pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -52,6 +55,9 @@ type ClusterMemberController struct {
 
 	masterNodeLister   corev1listers.NodeLister
 	masterNodeSelector labels.Selector
+	memberListerCtrl   *etcdmemberlistercontroller.EtcdMemberListerController
+	members            []etcdserverpb.Member
```

Comment:
this controller has two usage of `memberlist()`
- one uses to validate if a member is already part of the cluster 
- one iterates over the members and attempt to promote learners

Therefore i added both these fields and i update them on notifications from `etcdmemberlistercontroller` 

please let me know wdyt  cc @tjungblu @hasbro17